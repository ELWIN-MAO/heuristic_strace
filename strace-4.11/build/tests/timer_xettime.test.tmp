
mym_waitpid=103057

mym_waitpid=103057

mym_waitpid=103057

syscall_enter_mym_scno=59  sys_name=execve

mym_waitpid=103057

mym_waitpid=103057

syscall_exit_mym_scno=59  sys_name=execve

mym_waitpid=103057

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=103057

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=103057

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=103057

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=103057

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103057

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103057

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=103057

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=103057

syscall_enter_mym_scno=2  sys_name=open

mym_waitpid=103057

syscall_exit_mym_scno=2  sys_name=open

mym_waitpid=103057

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=103057

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=103057

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103057

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103057

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=103057

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=103057

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=103057

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=103057

syscall_enter_mym_scno=2  sys_name=open

mym_waitpid=103057

syscall_exit_mym_scno=2  sys_name=open

mym_waitpid=103057

syscall_enter_mym_scno=0  sys_name=read

mym_waitpid=103057

syscall_exit_mym_scno=0  sys_name=read

mym_waitpid=103057

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=103057

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=103057

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103057

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103057

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=103057

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=103057

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103057

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103057

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103057

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103057

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=103057

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=103057

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103057

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103057

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103057

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103057

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103057

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103057

syscall_enter_mym_scno=158  sys_name=arch_prctl

mym_waitpid=103057

syscall_exit_mym_scno=158  sys_name=arch_prctl

mym_waitpid=103057

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=103057

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=103057

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=103057

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=103057

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=103057

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=103057

syscall_enter_mym_scno=11  sys_name=munmap

mym_waitpid=103057

syscall_exit_mym_scno=11  sys_name=munmap

mym_waitpid=103057

syscall_enter_mym_scno=222  sys_name=timer_create
timer_create(CLOCK_MONOTONIC, {sigev_signo=0, sigev_notify=SIGEV_NONE}, 
mym_waitpid=103057

syscall_exit_mym_scno=222  sys_name=timer_create
[0]
mym_scall_rval=0
) = 0

mym_waitpid=103057

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=103057

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=103057

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=103057

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=103057

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=103057

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=103057

syscall_enter_mym_scno=223  sys_name=timer_settime
timer_settime(0, 0, {it_interval={233811169, 233811170}, it_value={233811171, 233811172}}, 
mym_waitpid=103057

syscall_exit_mym_scno=223  sys_name=timer_settime
{it_interval={0, 0}, it_value={0, 0}}
mym_scall_rval=0
) = 0

mym_waitpid=103057

syscall_enter_mym_scno=224  sys_name=timer_gettime
timer_gettime(0, 
mym_waitpid=103057

syscall_exit_mym_scno=224  sys_name=timer_gettime
{it_interval={233811169, 233811170}, it_value={233811171, 233693762}}
mym_scall_rval=0
) = 0

mym_waitpid=103057

syscall_enter_mym_scno=1  sys_name=write

mym_waitpid=103057

syscall_exit_mym_scno=1  sys_name=write

mym_waitpid=103057

syscall_enter_mym_scno=231  sys_name=exit_group

mym_waitpid=103057
+++ exited with 0 +++
