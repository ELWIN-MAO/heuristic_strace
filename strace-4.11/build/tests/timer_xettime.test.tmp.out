timer_create(CLOCK_MONOTONIC, {sigev_signo=0, sigev_notify=SIGEV_NONE}, [0]) = 0
timer_settime(0, 0, {it_interval={233811169, 233811170}, it_value={233811171, 233811172}}, {it_interval={0, 0}, it_value={0, 0}}) = 0
timer_gettime(0, {it_interval={233811169, 233811170}, it_value={233811171, 233693762}}) = 0
+++ exited with 0 +++
lo IP Address 127.0.0.1
ens33 IP Address 192.168.159.137
IP Address 127.0.0.1
IP Address 192.168.159.137
